### DISCUSSION TITLE
blog/graphql-error-handling-with-fp

### DISCUSSION THREAD

**Discussion by giscus** [the-guild-org/the-guild-website#1444] [Docs Discussions]
# blog/graphql-error-handling-with-fp

Handle unsafe APIs in a typesafe way with Typescript and Functional Programming.

https://the-guild.dev/blog/graphql-error-handling-with-fp

**Comment by Lonli-Lokli** [comment]
Problem with your proposal is that it adds nothing to most popular case when frontend developer cannot control backend ie cannot change graphql schema tlfor better error handling
